### DISCUSSION TITLE
Local License won't renew because of a Cloud Backup

### DISCUSSION THREAD

**Discussion by AlvinSaldanha** [ravendb/ravendb#13200] [Q&A] [answered]
I have a community version of RavenDB running locally. We use the cloud version for our dev/test/demo/prod environments. I had restored a cloud backup locally and the License has expired. 

Now new license key doesn't work as I have a cloud Backup.

And I cannot get inside raven Studio to delete the backup.

How can I renew the License?

![image](https://user-images.githubusercontent.com/20337663/144336010-ce8527a3-b176-46c4-854c-1ac6b422de14.png)

**Comment by garayx** [comment, marked as answer]
Hi

Try to use a developer license instead,

[https://ravendb.net/license/request/dev](https://ravendb.net/license/request/dev)
